packages/pangea-sdk/PangeaCyber.Net/AuthN/Clients/User.cs

```diff
@@ -49,54 +49,69 @@ public async Task<Response<UserCreateResult>> Create(UserCreateRequest request)
             return await DoPost<UserCreateResult>("/v2/user/create", request);
         }
 
-        /// <kind>method</kind>
         /// <summary>Delete a user by email address.</summary>
         /// <remarks>Delete User - Email</remarks>
         /// <operationid>authn_post_v2_user_delete 1</operationid>
-        /// <param name="email" type="string">An email address</param>
-        /// <returns>Response&lt;UserDeleteResult&gt;</returns>
+        /// <param name="email">An email address</param>
+        /// <returns>An empty object.</returns>
         /// <example>
         /// <code>
         /// await client.User.DeleteByEmail("joe.user@pangea.cloud");
         /// </code>
         /// </example>
-        public async Task<Response<UserDeleteResult>> DeleteByEmail(string email)
+        public async Task<Response<object>> DeleteByEmail(string email)
         {
             var request = new UserDeleteByEmailRequest(email);
-            return await DoPost<UserDeleteResult>("/v2/user/delete", request);
+            return await DoPost<object>("/v2/user/delete", request);
         }
 
-        /// <kind>method</kind>
         /// <summary>Delete a user by ID.</summary>
         /// <remarks>Delete User - ID</remarks>
         /// <operationid>authn_post_v2_user_delete 2</operationid>
-        /// <param name="id" type="string">The identity of a user or a service</param>
-        /// <returns>Response&lt;UserDeleteResult&gt;</returns>
+        /// <param name="id">The identity of a user or a service</param>
+        /// <returns>An empty object.</returns>
         /// <example>
         /// <code>
         /// await client.User.DeleteByID("pui_xpkhwpnz2cmegsws737xbsqnmnuwtbm5");
         /// </code>
         /// </example>
-        public async Task<Response<UserDeleteResult>> DeleteByID(string id)
+        public async Task<Response<object>> DeleteByID(string id)
         {
             var request = new UserDeleteByIDRequest(id);
-            return await DoPost<UserDeleteResult>("/v2/user/delete", request);
+            return await DoPost<object>("/v2/user/delete", request);
+        }
+
+        /// <summary>Delete a user by username.</summary>
+        /// <remarks>Delete User - Username</remarks>
+        /// <operationid>authn_post_v2_user_delete 3</operationid>
+        /// <param name="username">A username.</param>
+        /// <returns>An empty object.</returns>
+        /// <example>
+        /// <code>
+        /// await client.User.DeleteByUsername("foobar");
+        /// </code>
+        /// </example>
+        public async Task<Response<object>> DeleteByUsername(string username)
```

Comment:
I know this was going to happen, I should have done just a `Delete` method that receive a `request` object